<launch> 


  <node name="gps_msgs_package_node" pkg="gps_msgs_package" type="gps_msgs_package_node" output="screen" > 
     
          <param name="imu_yaw_offset"   value="0"/> -->
          <param name ="imu_topic" value="/vectornav/IMU"/>

      <remap from="/gps1/fix"     to="/ublox_gps/fix" />       
      
  </node>
</launch> 
